### PR TITLE
completed Task #5, and implemented viewbob_offset function

### PR DESCRIPTION
**Pull request by mistcurve**

### CODE REVIEW

**Comment by ryansteeters** [comment]
Good work, Jimmy. That method that you implemented will be useful for the next backlog item I will be working on.

**Review by n0dam — COMMENTED** [review]
Interesting way of manipulating variables I might try some of what you did here for the function I'm trying to use.

File: src/main.c [hunk 1]
```diff
@@ -2452,6 +2452,8 @@ void handle_movement(double dt) {
         float m = dt * 1.0;
         g->ortho = glfwGetKey(g->window, CRAFT_KEY_ORTHO) ? 64 : 0;
         g->fov = glfwGetKey(g->window, CRAFT_KEY_ZOOM) ? 15 : 65;
+        if (glfwGetKey(g->window, CRAFT_KEY_CROUCH)) viewBob_offSet(0);
```
Comment (jrichards1999) [inline src/main.c:2455]: Good use of descriptive variables here.  It makes it much easier to understand that this code correlates to a change in the view when the crouch key is pressed.

**Review by jrichards1999 — COMMENTED** [review]
Good work, Jimmy.  The code looks minimal and clean.  Especially like the use of descriptive variables to make the code easier to read.  My only recommendations are to add some doxygen comments explaining what's being done, along with some test code to prove that the code is being executed.

**Pull request merged**